opensourceleg/sensors/README.md

```diff
@@ -0,0 +1,133 @@
+﻿# [Actuators Base Library](./base.py) Guide
```

Comment:
forgot to update README here. Should update later

Reply:
Hey @Robin0265, what's the status on this?